subgraphs/yeti-finance/README.md

```diff
@@ -0,0 +1 @@
+Yeti Finance
```

Comment:
You should complete the README like the other subgraphs
e.g. https://github.com/messari/subgraphs/blob/master/subgraphs/liquity/README.md